ReactiveCocoa/DynamicProperty.swift

```diff
@@ -45,16 +45,42 @@ public final class DynamicProperty<Value>: MutablePropertyProtocol {
 		return cache.signal
 	}
 
-	/// Create a typed mutable view to the given key path of the given Objective-C object.	
+	internal init(object: NSObject, keyPath: String, cache: Property<Value>) {
+		self.object = object
+		self.keyPath = keyPath
+		self.cache = cache
+	}
+	
+	/// Create a typed mutable view to the given key path of the given Objective-C object.
 	/// The generic type `Value` can be any Swift type, and will be bridged to Objective-C
 	/// via `Any`.
 	///
 	/// - parameters:
 	///   - object: The Objective-C object to be observed.
 	///   - keyPath: The key path to observe.
-	public init(object: NSObject, keyPath: String) {
-		self.object = object
-		self.keyPath = keyPath
-		self.cache = Property(object: object, keyPath: keyPath)
+	public convenience init(object: NSObject, keyPath: String) {
+		self.init(object: object, keyPath: keyPath, cache: Property(object: object, keyPath: keyPath))
+	}
+}
+
+extension DynamicProperty where Value: OptionalProtocol {
+	
+	/// The current value of the property, as read and written using Key-Value
+	/// Coding.
+	public var value: Value {
```

Comment:
Not quite sure why this needs to be overriden.

Reply:
I don't know why it happens, but otherwise the `should write the underlying object` Testcase fails with:

> expected to be nil, got <<null>>

Reply:
I am a bit skeptical if this overload is called in a generic context (`MutablePropertyProtocol` specifically) though.

Reply:
Is ae79efb what you meant?

Reply:
Yep. Looks nice.